libs/designsystem/src/lib/components/tab-navigation/tab-navigation/tab-navigation.component.ts

```diff
@@ -0,0 +1,136 @@
+import {
```

Comment:
As I have hinted at before, I think the logic in this file is quite complex for tabbing between the individual buttons, and could benefit from a (hopefully simpler) approach with [roving tabindex](https://www.youtube.com/watch?v=uCIC2LNt0bk) if that fits the usecase.

But given the tight time constraints, and the fact that it seems to work, I think it would be better to follow up. Can you make an issue for that?